### PR TITLE
Update Owners File

### PR DESCRIPTION
**Pull request by dkeohane**
Update owners file which was quite out of date

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 - iamkirkbater
+- rogbas
 - ariellima
+- lisa
+- drpaneas
+- dastergon
+- dkeohane
 approvers:
 - fahlmant
 - rogbas
```
Comment (ArielLima) [inline OWNERS:12]: I think we don't need to bring rogbas from the a-a-0

**Review by fahlmant — COMMENTED** [review]
I'm not against this, but I'd like to ensure everyone in this list understands the shredder. Should we organize an informational meeting?

**Comment by ArielLima** [comment]
> I'm not against this, but I'd like to ensure everyone in this list understands the shredder. Should we organize an informational meeting?

@fahlmant I think we've already had this conversation and people agreed that matching the A-A-O would be the right move? I personally would've liked to be added to be added to the `approvers` list since I understand it fairly well but that's not important. Here is the convo: https://coreos.slack.com/archives/CNJPJSG3Z/p1624376764201300

**Comment by fahlmant** [comment]
We can match the list once everyone has the knowledge needed to become approves and reviewers.

**Comment by lisa** [comment]
not sure why i put a hold on this

/unhold

**Review by fahlmant — APPROVED** [review]
/lgtm

**Pull request merged**